wiki/en/misc/1-index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Jamulus ‒ Play music online. With friends. For free."
+lang: "en"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Music"
+ldjSlogan: "Play music online. With friends. For free."
+ldjCreator: "Volker Fischer and contributors"
+metadescription: "Jamulus is free and open source software that lets musicians perform music, rehearse or jam together, in real time over the Internet."
+mAltProgIcon: "Jamulus icon"
+mTSlogan: "Play music online. With friends. For free."
+mTGetStartedNow: "Get started now!"
+mTDownloadNow: 'Download now for'
+mTPlatformsAnd: 'and'
+mTOtherPlatforms: 'other platforms'
+---
+[//]: # Markdown sections should not be indented
+<div class="fx-row fx-row-center-xs" id="firstrow">
+  <div class="fx-col-100-xs">
+    <div itemprop="abstract">
+      <h2>What is Jamulus?</h2>
+<div markdown="1">
+Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. [Download it here!](wiki/Getting-Started)
+</div>
+    </div>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+  <div class="fx-col-100-xs">
+    <a href="wiki/Getting-Started">
+      <img alt="Jamulus Banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
```

Comment:
```suggestion
      <img alt="Jamulus banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
```

Reply:
I'd rather you proposed these changes in another PR. This one is to fix the segmentation issue.

Reply:
It won't hurt my feelings if you ignore my proposed improvements, though I hope your view changes.

Reply:
@mcfnord your proposals are fine, I agree with them - I just think they belong in a separate PR.

Reply:
Does accepting the suggestions interfere with your workflow? I know they create commit messages that I don't know how to squash yet in my own PRs.

Reply:
@mcfnord you can try https://www.git-tower.com/learn/git/faq/git-squash (maybe GitHub desktop/GitKraken, ... helps you? They're graphical tools. Also see https://blog.devart.com/best-git-gui-clients-for-windows.html)

Reply:
I will study those links, and will learn to squash. I'm asking why ignotus666 won't accept these suggestions in this PR. I drive-by suggest every chance I get. I want to understand what prevents acceptance in some instances.

Reply:
If for whatever reason we want to find what PR a change happened in, the PR title should give us a good clue about it. Your proposed changes have nothing to do with the purpose of this PR, which is purely to solve a technical issue. Maybe I'm wrong, but I think the policy here is to keep PRs and issues on topic and not have them branch out in all directions.

Reply:
> Maybe I'm wrong, but I think the policy here is to keep PRs and issues on topic and not have them branch out in all directions.

Yes. this makes discussion more focussed.

Reply:
It is threaded tho. It's an empty seat and a free ride into town.